core/arch/arm/kernel/asm-defines.c

```diff
@@ -127,4 +128,10 @@ DEFINES
 	       offsetof(struct boot_embdata, reloc_offset));
 	DEFINE(BOOT_EMBDATA_RELOC_LEN,
 	       offsetof(struct boot_embdata, reloc_len));
+#ifdef ARM64
```

Comment:
needs a specific section: inline comment above (`/* struct boot_embdata */`) is not accurate.

Can remove the line break, each fit in a single line.